lib/user.js

```diff
@@ -9,20 +9,26 @@ const ensureUser = async function (claims) {
     const queryResult = await query(`
     PREFIX foaf: <http://xmlns.com/foaf/0.1/>
     PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX adms: <http://www.w3.org/ns/adms#>
 
-    SELECT ?person ?firstName ?familyName
+    SELECT ?person ?firstName ?familyName ?status
     WHERE {
       GRAPH <${USERS_GRAPH}> {
         ?person a foaf:Person ;
           dct:identifier ${sparqlEscapeString(userIdentifier)} .
         OPTIONAL { ?person foaf:firstName ?firstName . }
         OPTIONAL { ?person foaf:familyName ?familyName . }
+        OPTIONAL { ?person adms:status ?status }
       }
     }`);
 
     if (queryResult.results.bindings.length) {
       const result = queryResult.results.bindings[0];
       const personUri = result.person.value;
+      const status = result.status.value;
+      if (status === ACCESS_BLOCKED_STATUS_URI) {
+        throw new Error(`The user has been blocked and does not have access.`)
```

Comment:
Custom error gooien zodat die in app.js opgevangen kan worden en 403 Forbidden kan returnen.